assignment/README.md

```diff
@@ -55,6 +125,11 @@ are hard to use and think about. Global variables are a strategy for
 coping with that difficulty by breaking data out of the asynchronous
 context.
 
+- The most difficult part may be an interactive chart that shows the
```

Comment:
@AstoriaYU 

Next week @moradology will be going over chart.js.

You can get a head start looking at the documentation:
https://chartjs.org/samples/latest/

Double check which one you might want to use, but I think if you can pull this off well it's going to make this a strong project.